src/app/pages/jails/jail-list/jail-list.component.ts

```diff
@@ -116,20 +116,26 @@ export class JailListComponent implements OnInit {
       enable: true,
       ttpos: "above",
       onClick: (selected) => {
-        let selectedJails = this.getSelectedNames(selected);
-        this.loader.open();
+        const selectedJails = this.getSelectedNames(selected);
+        this.snackBar.open(T('Update selected plugins started.'), 'close', { duration: 5000 });
         this.entityList.busy =
           this.ws.job('core.bulk', ["jail.update_to_latest_patch", selectedJails]).subscribe(
             (res) => {
-              this.loader.close();
-              if (res.state == 'SUCCESS') {
-                this.snackBar.open(T("Selected Jail(s) updated successfully."), T("Close"), { duration: 5000 });
+              let message = "";
+              for (let i = 0; i < res.result.length; i++) {
+                if (res.result[i].error != null) {
+                  message = message + '<li>' + selectedJails[i] + ': ' + res.result[i].error + '</li>';
+                }
+              }
+              if (message === "") {
+                this.snackBar.open(T('Update selected jails succeeded.'), 'close', { duration: 5000 });
               } else {
-                new EntityUtils().handleWSError(this.entityList, res, this.dialogService);
+                message = '<ul>' + message + '</ul>';
+                this.dialogService.errorReport(T('Update following selected jails failed.'), message);
               }
             },
             (res) => {
-              this.loader.close();
+              this.snackBar.open(T('Update selected jails failed.'), 'close', { duration: 5000 });
```

Comment:
As above, put "Failed" first.  And it would help the user to know whether it was all or some of those jails.  So this should probably be a loop, showing them a message for each jail that failed to update.
However, we really should avoid use of the aggravating snackbar, especially if it's going to show the user a bunch of them where each has to be clicked.

Reply:
this message is for general update call. If one or more selected jails update failed, it will list them in a dialog (see line 134).